mw-webapp/src/component/table/Table.tsx

```diff
@@ -9,7 +9,7 @@ export const Table = () => {
   const [data, setData] = useState<DayReport[]>([]);
 
   useEffect(() => {
-    DayReportService.onValueFromRealTimeDb(setData);
+    DayReportService.getDayReports(setData);
```

Comment:
Usually getDayReports should return day reports instead of do nothing and call some callback inside

Reply:
Change

Reply:
Change